service/service.go

```diff
@@ -175,7 +175,7 @@ func New(config Config) (*Service, error) {
 		informerConfig.BackOff = backoff.NewExponentialBackOff()
 		informerConfig.WatcherFactory = newWatcherFactory
 
-		informerConfig.RateWait = time.Second * 10
+		informerConfig.RateWait = time.Second * 1
```

Comment:
For testing purposes now, will probably change soon.

Reply:
As I already said in Slack. You do not need to set any rate wait here. The default is 0 and we only use 10 seconds as magic number without any background for other computation and/or network heavy processing. Since we filter 95% of the pods anyway we could just go with the default value. I will think about filtering support within our informers to make work like you want to do it more easy and nice.

Reply:
I missed your message in slack, removed it now.